src/core/gosling.schema.ts

```diff
@@ -209,6 +209,50 @@ export interface CommonTrackDef extends CommonViewDef {
 export interface DataTrack extends CommonTrackDef {
     data: DataDeep;
 }
+/**
+ * A placeholder track. In contrast to other tracks, this track does not display any data. Instead it provides
+ * empty space for third party tools to display their data on top of.
+ */
+export interface DummyTrack
+    extends Pick<
+        CommonTrackDef,
+        | 'width'
+        | 'height'
+        | 'id'
+        | 'title'
+        | '_invalidTrack'
+        | 'overlayOnPreviousTrack'
+        | 'orientation'
+        | 'layout'
+        | 'static'
+        | 'assembly'
+    > {
+    /** Used to specify the dummy track */
+    type: 'dummy-track';
+    /** Defines how the track is styled */
+    style?: DummyTrackStyle;
+    /** Dummy track layout must be linear */
+    layout?: 'linear';
+    /** Dummy track can only be static */
+    static?: true;
+    /** Dummy track cannot be zoomed in on */
+    zoomLimits?: [null, null];
+    /** No assemblies are associated with a dummy track */
+    assembly?: 'unknown';
+}
```

Comment:
Rather than just extending `CommonTrackDef`, I tried making making the properties used explicit so users won't be confused. Some of them like `layout`, `static`, and `zoomLimits` I kept in there they're heavily used in `spec-preprocess.ts` and I would have to significantly refactor it to not have them. Let me know if you think there would be a better way to do this.